### PR TITLE
SAAS-178 apply pmm patches

### PR DESCRIPTION
**Pull request by atymchuk**

### CODE REVIEW

File: public/app/features/dashboard/components/ShareModal/ShareSnapshot.tsx [hunk 1]
```diff
@@ -212,13 +213,34 @@ export class ShareSnapshot extends PureComponent<Props, State> {
...
+              <li>
+                <strong>Expire:</strong> The time before snapshot expires. Configure lower if required. Percona
+                automatically purge shared dashboards after 90 days.
+              </li>
+              <li>
+                <strong>Timeout (seconds):</strong> Time the dashboard takes to load before the snapshot is generated.
+              </li>
+            </ul>
+          </p>
+          +{' '}
```
Comment (nicolalamacchia) [inline public/app/features/dashboard/components/ShareModal/ShareSnapshot.tsx:239]: This looks like a typo :)

File: pkg/services/licensing/oss.go [hunk 2]
```diff
@@ -38,16 +38,16 @@ func (l *OSSLicensingService) LicenseURL(user *models.SignedInUser) string {
...
+		// for _, node := range indexData.NavTree {
+		// 	if node.Id == "admin" {
+		// 		node.Children = append(node.Children, &dtos.NavLink{
+		// 			Text: "Upgrade",
+		// 			Id:   "upgrading",
+		// 			Url:  l.LicenseURL(req.SignedInUser),
+		// 			Icon: "unlock",
+		// 		})
+		// 	}
+		// }
```
Comment (nicolalamacchia) [inline pkg/services/licensing/oss.go:50]: This comment should be removed altogether

File: public/views/error-template.html [hunk 3]
```diff
@@ -12,7 +12,7 @@
 
     <link rel="stylesheet" href="public/build/grafana.[[ .Theme ]].<%= webpack.hash %>.css">
 
-    <link rel="icon" type="image/png" href="public/img/fav32.png">
+    <link rel="icon" type="image/x-icon" href="public/img/percona-favicon.ico?cb=1584890955588">
```
Comment (nicolalamacchia) [inline public/views/error-template.html:15]: Is there a reason to keep the cache bump tag for the icon?

File: public/views/index-template.html [hunk 4]
```diff
@@ -30,7 +30,7 @@
       crossorigin
     />
 
-    <link rel="icon" type="image/png" href="[[.FavIcon]]" />
+    <link rel="icon" type="image/x-icon" href="public/img/percona-favicon.ico?cb=1584890955588">
```
Comment (nicolalamacchia) [inline public/views/index-template.html:33]: Is there a reason to keep the cache bump tag for the icon?

File: public/views/error-template.html (hunk 3 above)
Comment (atymchuk) [reply public/views/error-template.html:15]: Yes, the old builds contained a pixelated icon with the same name, so this was to make the browser load a new icon versus the cached one.

File: pkg/services/licensing/oss.go (hunk 2 above)
Comment (atymchuk) [reply pkg/services/licensing/oss.go]: Done

File: public/app/features/dashboard/components/ShareModal/ShareSnapshot.tsx (hunk 1 above)
Comment (atymchuk) [reply public/app/features/dashboard/components/ShareModal/ShareSnapshot.tsx]: usual copy-paste stuff )

**Pull request merged**